src/cube_playtest.js

```diff
@@ -96,12 +97,9 @@ const DecksCard = ({ decks, cubeID, ...props }) => (
     <CardHeader>
       <CardTitleH5>Recent Decks</CardTitleH5>
     </CardHeader>
-    <CardBody>
+    <CardBody className="p-0">
       {decks.map(deck =>
-        <Fragment key={deck._id}>
-          <a href={`/cube/deck/${deck._id}`}>{deck.name}</a>
-          <br />
-        </Fragment>
+        <DeckPreview deck={deck}/>
```

Comment:
This needs a `key` prop.
```suggestion
        <DeckPreview key={deck._id} deck={deck} />
```